feat: Add support for `hana` (#463)
succinctlabs/op-succinct: book/validity/experimental/experimental.md

# Experimental Features

> This section covers experimental features in the validity system.

Reviewer: ```suggestion
This section covers experimental features for OP Succinct (Validity).
```

## Celestia DA

> The `op-succinct-celestia` service monitors the state of the L2 chain with Celestia DA enabled, requests proofs from the Succinct Prover Network[https://docs.succinct.xyz/docs/network/introduction] and submits them to the L1.

Reviewer: ```suggestion
The `op-succinct-celestia` service monitors the state of an OP Stack chain with Celestia DA enabled, requests proofs from the Succinct Prover Network[https://docs.succinct.xyz/docs/network/introduction] and submits them to the L1.
```
